.github/workflows/release.yml

```diff
@@ -13,7 +14,58 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
+  test-kubo:
+    name: 'test:kubo'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: L1-node
+
+      - name: Checkout Kubo fork
+        uses: actions/checkout@v3
+        with:
+          repository: filecoin-saturn/kubo
+          ref: refs/heads/saturn
+          path: kubo
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Install Kubo gateway test dependencies
+        run: |
+          cd $GITHUB_WORKSPACE/kubo
+          make test_sharness_deps
+
+      - name: Run Kubo gateway tests
+        run: |
+          export L1_NODE_REPO_PATH=$GITHUB_WORKSPACE/L1-node
+          #export L1_NODE_LOG_FILE=/tmp/node.log
+
+          cd $GITHUB_WORKSPACE/kubo/test/sharness
+
+          for file in $(find .  -maxdepth 1 -name 't011*-gateway*.sh' -type f); do
+            echo
+            if [[ $file == *"t0111"* || $file == *"t0114"* ]]; then
+              echo Skip $file
+              continue
+            fi
+            echo Running $file
+            bash $file
+          done
+
+      # - name: Print L1 logs
+      #   if: always()
+      #   run: cat /tmp/node.log
```

Comment:
instead of commenting out these, we could also always attach the logs to the workflow run (I don't think there's harm in doing it every time), using 

```yaml
name: Attach produced packages to Github Action
        uses: actions/upload-artifact@v2
        with:
          name: kubo-logs
          path: /tmp/node.log
```

Reply:
True. I could also just uncomment it? We don't really need these logs in long term storage. If someone needs to view the logs it'll be because the workflow failed and they want to view the logs immediately instead of later.

Reply:
Sure!